### PR TITLE
[MOB-3453] Update Spruce sensor icon

### PR DESCRIPTION
**Pull request by jackchi**
Can you reach out to Nathan Cauffman to see if he wants this icon?

### CODE REVIEW

File: devicetypes/plaidsystems/spruce-sensor.src/spruce-sensor.groovy [hunk 1]
```diff
@@ -73,7 +73,7 @@ metadata {
 					[value: 64, color: "#44B621"],
 					[value: 80, color: "#3D79D9"],
 					[value: 96, color: "#0A50C2"]
-				]
+				], icon: 'http://www.plaidsystems.com/smartthings/st_spruce_leaf_225_top.png'
```
Comment (workingmonk) [inline devicetypes/plaidsystems/spruce-sensor.src/spruce-sensor.groovy]: can we use an existing humidity icon? if they change the link, this won't look pretty on the DTH

Comment (jackchi) [reply devicetypes/plaidsystems/spruce-sensor.src/spruce-sensor.groovy]: Yeah Nathan (Spruce device maker) confirmed he wanted this. I'll let him make the call.

**Comment by workingmonk** [comment]
cc @dkirker @tylerlange

File: devicetypes/plaidsystems/spruce-sensor.src/spruce-sensor.groovy (hunk 1 above)
Comment (natec007) [reply devicetypes/plaidsystems/spruce-sensor.src/spruce-sensor.groovy]: Which humidity icon would you use? Do you have a link so I can see it?

Comment (dkirker) [reply devicetypes/plaidsystems/spruce-sensor.src/spruce-sensor.groovy]: There is a list of icons gathered by someone here: http://scripts.3dgo.net/smartthings/icons/
Someone else has also created: https://cdn.rawgit.com/JZ-SmartThings/SmartThings/master/Icons/ST-Icons.html

A good choice might be st.Outdoor.outdoor12 or st.Weather.weather12

That said, I think keeping custom icon assets available if used would fall under the WWST agreement requirement?

Anyway, if the Spruce logo is used, my recommendation is to make sure that it has the same aspect ratio and whitespace, as well as the same color, as the other icons.

Comment (natec007) [reply devicetypes/plaidsystems/spruce-sensor.src/spruce-sensor.groovy]: st.Weather.weather12 is fine
Thanks

Comment (workingmonk) [reply devicetypes/plaidsystems/spruce-sensor.src/spruce-sensor.groovy]: @jackchi can you change the icon to what @natec007 agreed to?

**Pull request merged**